bridgetown-core/lib/bridgetown-core/converters/erb_templates.rb

```diff
@@ -131,7 +131,7 @@ def convert(content, convertible)
       end
 
       def matches(ext, convertible)
-        if convertible.data[:template_engine] == "erb" ||
+        if convertible.data[:template_engine].to_s == "erb" ||
             (convertible.data[:template_engine].nil? &&
               @config[:template_engine] == "erb")
```

Comment:
should I add `to_s` here too?

Reply:
I think you meant @jaredcwhite 😊

Reply:
> should I add `to_s` here too?

@JuanVqz I'd suggest a broad search for￼ "erb", might need to update a few places.￼

Probably have the same issue with "liquid" as well.￼